google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1.rb

```diff
@@ -18,97 +18,97 @@
-    #
-    # ### Preview
-    # #### VideoIntelligenceServiceClient
-    # ```rb
-    # require "google/cloud/video_intelligence/v1"
-    #
-    # video_intelligence_service_client = Google::Cloud::VideoIntelligence::V1.new
-    # input_uri = "gs://demomaker/cat.mp4"
-    # features_element = :LABEL_DETECTION
-    # features = [features_element]
-    #
-    # # Register a callback during the method call.
-    # operation = video_intelligence_service_client.annotate_video(input_uri: input_uri, features: features) do |op|
-    #   raise op.results.message if op.error?
-    #   op_results = op.results
-    #   # Process the results.
-    #
-    #   metadata = op.metadata
-    #   # Process the metadata.
-    # end
-    #
-    # # Or use the return value to register a callback.
-    # operation.on_done do |op|
-    #   raise op.results.message if op.error?
-    #   op_results = op.results
-    #   # Process the results.
-    #
-    #   metadata = op.metadata
-    #   # Process the metadata.
-    # end
-    #
-    # # Manually reload the operation.
-    # operation.reload!
-    #
-    # # Or block until the operation completes, triggering callbacks on
-    # # completion.
-    # operation.wait_until_done!
-    # ```
-    #
-    # ### Next Steps
-    # - Read the [Cloud Video Intelligence API Product documentation][Product Documentation]
-    #   to learn more about the product and see How-to Guides.
-    # - View this [repository's main README](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/README.md)
-    #   to see the full list of Cloud APIs that we cover.
-    #
-    # [Product Documentation]: https://cloud.google.com/video-intelligence
-    #
-    # ## Enabling Logging
-    #
-    # To enable logging for this library, set the logger for the underlying [gRPC](https://github.com/grpc/grpc/tree/master/src/ruby) library.
-    # The logger that you set may be a Ruby stdlib [`Logger`](https://ruby-doc.org/stdlib-2.5.0/libdoc/logger/rdoc/Logger.html) as shown below,
-    # or a [`Google::Cloud::Logging::Logger`](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-logging/latest/google/cloud/logging/logger)
-    # that will write logs to [Stackdriver Logging](https://cloud.google.com/logging/). See [grpc/logconfig.rb](https://github.com/grpc/grpc/blob/master/src/ruby/lib/grpc/logconfig.rb)
-    # and the gRPC [spec_helper.rb](https://github.com/grpc/grpc/blob/master/src/ruby/spec/spec_helper.rb) for additional information.
-    #
-    # Configuring a Ruby stdlib logger:
-    #
-    # ```ruby
-    # require "logger"
-    #
-    # module MyLogger
-    #   LOGGER = Logger.new $stderr, level: Logger::WARN
-    #   def logger
-    #     LOGGER
-    #   end
-    # end
-    #
-    # # Define a gRPC module-level logger method before grpc/logconfig.rb loads.
-    # module GRPC
-    #   extend MyLogger
-    # end
-    # ```
-    #
     module VideoIntelligence
+      # rubocop:disable LineLength
```

Comment:
So happy to see this change! Been working on the HTML docs and not liking the inconsistencies in where the GAPIC files place documentation.